File: C/Assignments/Assignment 3/question_3.c
/*Q3 Sum of numbers in given range.
Find sum of numbers from start to end.
Input: start = 1, end = 5
Output: 15*/

#include<stdio.h>
void main(){
    int start, end, sum=0;

    printf("Enter start and end: ");
    scanf("%d %d", &start, &end);

    while(start<=end){
        sum+=start;
        start++;
    }
    printf("Sum of numbers in range: %d", sum);
}

File: CtoJava/src/Assignment_4/question_3.java
package Assignment_4;

public class question_3 {
	//Q3. Print perfect numbers in the given range 1 to n.

	
	public static void main(String args[]){

	    int j, n=100, sum;

	    

	    for(int i=1; i<=n; i++){
	        sum=0;
	        j=1;
	        while(j<i){
	            if(i%j==0)
	                sum += j;
	                j++;
	        }

	        if(sum==i)
	            System.out.printf("%d is perfect number\n", i);
	        
	    }
	}
}
